modules/public/logic.py

```diff
@@ -160,6 +166,21 @@ def build_country_info(users_by_country, display_number_users=False):
     return country_data
 
 
+def build_os_info(series):
```

Comment:
Would it make sense to add some unit tests for this, to make sure we are transforming the data in an expected way?

Reply:
and add docstring please

Reply:
Added